### PR TITLE
fix for high cpu usage

### PR DESCRIPTION
**Pull request by mistu4u**
### What does this do?
Fix high cpu usage while using gocron in wait mode. Reproducible code link: https://gist.github.com/mistu4u/49fd20dae3d5851ca7421c88b4eecd2e

### Which issue(s) does this PR fix/relate to?
https://github.com/go-co-op/gocron/issues/333

### Did you document any new/modified functionality?

- [ ] Updated `example_test.go`
- [ ] Updated `README.md`

### CODE REVIEW

**Comment by mistu4u** [comment]
@JohnRoesler Hi John, could you please take a look at this PR? In wait mode, we were facing 100% cpu usage in our project, this fix solves the problem. I see it was already reported thru bug https://github.com/go-co-op/gocron/issues/333

**Comment by JohnRoesler** [comment]
@mistu4u can you please revert the changes to the workflow files? 

Looks like a test is failing, i'll look into it

**Comment by mistu4u** [comment]
> @mistu4u can you please revert the changes to the workflow files?
> 
> Looks like a test is failing, i'll look into it
Sure John. I reverted the file.

File: .github/workflows/file_formatting.yml [hunk 1]
```diff
@@ -16,4 +16,4 @@ jobs:
         uses: actions/checkout@v3
       - name: verify example_test.go
         run: |
-          grep "^func " example_test.go | sort -C
+          grep "^func " example_test.go | sort -C
```
Comment (JohnRoesler) [inline .github/workflows/file_formatting.yml:19]:
```suggestion
          grep "^func " example_test.go | sort -C

```

**Pull request merged**